src/kili/adapters/kili_api_gateway/organization/operations.py

```diff
@@ -0,0 +1,74 @@
+"""Collection of Organization's related GraphQL queries and mutations."""
+
+ORGANIZATION_FRAGMENT = """
+id
+"""
```

Comment:
in other gateways, the fragment is not hard-coded

these operations can take a `fragment: str` parameter, and the gateway methods take a `fields` parameter: https://github.com/kili-technology/kili-python-sdk/blob/e3c7fabdff4d588936e6161580e67853ce4dae03/src/kili/adapters/kili_api_gateway/project/__init__.py#L100

so here, gateway methods update_organization and create_organization should build a fragment from the fields, and pass the fragment to the operations.py methods